lib/utils/idleState.ts

```diff
@@ -64,41 +65,30 @@ const USER_EVENTS = ['mousemove', 'keydown', 'wheel', 'mousedown', 'touchstart',
 // list of user events to ignore when window hasn't focus as string, i.e. 'mousemove touchmove'
 const IGNORED_USER_EVENTS = 'mousemove';
 
-const stateProperty = Symbol('state');
-
-const setState = (idleState: IdleState, state: 'active' | 'idle') => {
-  if (idleState[stateProperty] !== state) {
-    idleState[stateProperty] = state;
-    idleState.observers && idleState.observers.call(state);
-  }
-};
 class IdleState {
   refcount?: number;
   timeout?: number;
```

Comment:
I think the `timeout` property can be cleaned up, given it's been removed from the constructor.
```suggestion
```

Reply:
✅